src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts

```diff
@@ -26,12 +38,24 @@ export const TABULAR_MODIFICATION_FIELDS: TabularModificationFields = {
         'lowVoltageLimit',
         'highVoltageLimit',
     ],
+    TWO_WINDINGS_TRANSFORMER: [
+        EQUIPMENT_ID,
```

Comment:
Can't we also use constants for voltage_level, load and generator for consistency ?

Reply:
We should indeed, I don't know why they didn't do it

Reply:
The thing is we can't "clean" the code in US PR, and some peoples are working on these US.
We should create a TS to clean all this IMO.